### ISSUE TITLE
Add an option to disable SVGO completely

### ISSUE THREAD

**Issue by andreasvirkus** [visualfanatic/vue-svg-loader#31]
Hi,

Would it be possible to add an option to remove SVGO's optimisations from the loader completely?
Currently SVGO manages to strip my style tags, even when I set `removeStyleElement: false` (it strips the style tag from `<defs>` even after I've set every svgo plugin to `false`) 😕

**Comment by sam3d** [comment]
Literally having exactly the same problem and came here to see if anybody else had as well.

Gonna take a look through the source and see if I can make a PR.

**Comment by andreasvirkus** [comment]
@sam3d I created a custom SVG loader and it turns out Vue's `vue-template-compiler` is the issue. See a relevant comment thread: 
https://forum.vuejs.org/t/about-using-style-tags-inside-templates/7275

I've just changed my SVG structure to use attributes where possible and inline the styles where not. 😕

**Comment by visualfanatic** [comment]
@andreasvirkus disabling SVGO is now included in `0.10.0`, just pass `svgo: false` to loader options.

**Comment by evandrosevergnini** [comment]
> @andreasvirkus disabling SVGO is now included in `0.10.0`, just pass `svgo: false` to loader options.

Thanks!
Just for example, I used this way in 'vue.config.js' and it worked:

```´
chainWebpack: config => {
    const svgRule = config.module.rule('svg')
    svgRule.uses.clear()
    svgRule.use('vue-svg-loader').loader('vue-svg-loader').options({svgo: false})
},
```
